vue/sbc-common-components/src/components/SbcAuthenticationOptionsDialog.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <v-dialog v-model="showModal" max-width="850px" persistent>
+    <v-card class="pa-4">
+      <v-container class="view-container">
```

Comment:
There is a SbcAuthenticationOptions.vue component which has the same design and view of these. Can we leverage that to this, rather than repeating the code here?

Reply:
Hmm yea, it is very similar. After discussing with Jeremy we thought perhaps it would be better to create a new component specific to this scenario (dialog) instead of refactoring/complicating the existing full view component. 
Would you prefer to refactor the existing one?

Reply:
nope, refactoring wont be a good idea.
but we can include 'SbcAuthenticationOptions' as a child component here.
and include in the html inside the dialog container like <SbcAuthenticationOptions></SbcAuthenticationOptions >. and extend those functionalities to this modal popup.

Reply:
Ok, a sub template? With a prop determining which template to use?

Reply:
It might make sense to make the dialog a sub template of the full view, no?

Reply:
no, the other way will be better.

SbcAuthenticationOptions as a sub template in this Dialog popup component.

we can define the same prop names in the dialog component to pass the values between the components